src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/network_rule.py

```diff
@@ -0,0 +1,93 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+
+from azure.mgmt.containerregistry.v2018_09_01.models import (
```

Comment:
You cannot import this with a hard coded version. You must update the module to use a multi-API-versioned SDK.

Reply:
@tjprescott We had offline discussions with @yugangw-msft We will clean this up plus remove deprecated legacy commands as a separate PR, tracked by this issue https://github.com/Azure/azure-cli/issues/8265

Reply:
Sounds good @djyou.  AKS team is following the same approach.